src/data/registry.xml

```diff
@@ -203,15 +203,15 @@
       <ic_file_input_names>zi state_zi</ic_file_input_names>
     </variable>
     <variable local_name="te_ini"
-              standard_name="vertically_integrated_energies_of_initial_state_in_cam"
+              standard_name="vertically_integrated_total_energy_of_initial_state"
               units="J m-2"
               type="real" kind="kind_phys"
               allocatable="pointer">
       <dimensions>horizontal_dimension</dimensions>
       <ic_file_input_names>te_ini state_te_ini</ic_file_input_names>
     </variable>
     <variable local_name="te_cur"
-              standard_name="vertically_integrated_energies_of_current_state_in_cam"
+              standard_name="vertically_integrated_total_energy_of_current_state"
```

Comment:
Same, should be `vertically_integrated_total_energy_of_current_state_using_physics_energy_formula`

Reply:
updated!